tests/system_health_test.py

```diff
@@ -0,0 +1,330 @@
+        else:
+            state = MockerManager.STATE_RUNNING
+            stats = {}
+        MockerManager.counter += 1
+
+        return state, stats
+
+class MockerChassis(object):
+    counter = 0
+
+    def initizalize_system_led(self):
+        return
+
+    def get_status_led(self):
+        if MockerChassis.counter == 1:
+            MockerChassis.counter += 1
+            return "green"
+        else:
+            MockerChassis.counter += 1
+            return "red"
+
+import show.main as show
+
+class TestHealth(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        os.environ["PATH"] += os.pathsep + scripts_path
+        os.environ["UTILITIES_UNIT_TESTING"] = "1"
+
+    def test_health_summary(self):
+        runner = CliRunner()
+        result = runner.invoke(show.cli.commands["system-health"].commands["summary"])
+        click.echo(result.output)
+        expected = """System health configuration file not found, exit...
+"""
```

Comment:
Suggest to reformat docstring for readability:

```python
        expected = """\
System health configuration file not found, exit...
"""
```

Reply:
Fixed.